src/component/todoList/homeTodoList/HomeTodoList.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import CompletedButton from "../../filterButton/completed/Completed";
+import CompletedTodoList from "../completedTodoList/CompletedTodoList";
+import ActiveTodoList from "../activeTodoList/ActiveTodoList";
+import AllTodoList from "../allTodoList/AllTodoList";
+import AscTodoList from "../ascTodoList/AscTodoList";
+import DscTodoList from "../dscTodoList/DscTodoList";
+import {
+  Button,
+  Checkbox,
+  AppBar,
+  Toolbar,
+  IconButton,
+  Typography,
+  CardContent,
+  Card,
+} from "@material-ui/core";
+import DscButton from "../../filterButton/dscButton/dscButton";
+import AscButton from "../../filterButton/ascButton/AscButton";
+import AllButton from "../../filterButton/allButton/AllButton";
+import ActiveListButton from "../../filterButton/activeList/ActiveList";
+import { makeStyles } from "@material-ui/core/styles";
+
+function TodoList(props) {
+  const useStyles = makeStyles((theme) => ({
+    todolistItems: {
+      marginTop: "20px",
+    },
+    todolistButtons: {
+      marginTop: "20px",
+    },
+    todoButtons: {
+      paddingLeft: "20px",
+      marginLeft: "20px",
+      margin: theme.spacing(1),
+    },
+  }));
+  const classes = useStyles();
+
+  return (
+    <div className={classes.todolistItems}>
+      {props.formState.buttonStatus == "all" ? (
+        <AllTodoList
+          formStates={props.formState}
+          handleChecked={(id) => props.handleChecked(id)}
+          handleDeleteListItem={(id) => props.handleDeleteListItem(id)}
+        />
+      ) : props.formState.buttonStatus == "completed" ? (
+        <CompletedTodoList
```

Comment:
this structure is also fully wrong.

why do we have so many different view for the same thing. 

 we should just changing data and have a single view

Reply:
ok sir